src/commands/moderation/ban.ts

```diff
@@ -81,6 +81,12 @@ const failedBanEmbed = (user: User, error: any) => makeEmbed({
     ],
 });
 
+const unknownUserEmbed = (userId: string) => makeEmbed({
+    title: 'Failed to Ban User - Unknown user',
+    color: Colors.Red,
+    description: `Can not ban an unknown user ${userId}. They are not part of this server.`,
```

Comment:
This isn't always when a user isn't in the server. A mod could have grabbed the messageID by accident etc.

Maybe worth a clarification like 'please check the user ID, if it is correct they are not part of the server' but written better haha!

Reply:
Done! Thanks for the feedback